bpm/bonita-api/bonita-common-api-impl/src/main/java/org/bonitasoft/engine/bpm/process/impl/ProcessDefinitionBuilder.java

```diff
@@ -507,7 +507,7 @@ private void validateNonInterruptingBoundaryEvent(final BoundaryEventDefinition
     private void validateBoundaryOutgoingTransitions(final BoundaryEventDefinition boundaryEvent) {
         for (final TransitionDefinition transition : boundaryEvent.getOutgoingTransitions()) {
             if (transition.getCondition() != null) {
-                designErrors.add("A boundary event must have not inconditional transitions: " + transition.getSource() + "->" + transition.getTarget());
+                designErrors.add("A boundary event must have not unconditional transitions: " + transition.getSource() + "->" + transition.getTarget());
```

Comment:
As far as I understand, message should be the contrary: "A boundary event must have no conditional transitions". no?